XiEditor/Main.storyboard

```diff
@@ -184,72 +184,172 @@ Gw
             <objects>
                 <viewController storyboardIdentifier="Find View Controller" id="lKE-xV-0EG" customClass="FindViewController" customModule="XiEditor" customModuleProvider="target" sceneMemberID="viewController">
                     <visualEffectView key="view" blendingMode="behindWindow" material="appearanceBased" state="inactive" id="NrM-ac-2ZW">
-                        <rect key="frame" x="0.0" y="0.0" width="480" height="38"/>
+                        <rect key="frame" x="0.0" y="0.0" width="1559" height="30"/>
```

Comment:
this is the nitiest of nits, but when I open the storyboard the find view _is_ really wide. On my laptop in particular this is kind of annoying; I can't fit the whole view on the screen.